src/SPTLikelihoods.jl

```diff
@@ -3,27 +3,38 @@ import Base.@kwdef
 using Artifacts
 using NPZ
 
+
+const tSZ_template = Vector{Float64}(undef, 3200)
```

Comment:
Before these weren't const so the functions using them were type unstable. I'm not sure there's a simper way to do it but maybe.